xsuite-lightsimulnet/src/handleAdmin.go

```diff
@@ -158,6 +231,16 @@ type RawAccount struct {
 	Owner					*string
 }
 
+type UpdateRawAccount struct {
```

Comment:
Let's make `Nonce` and `Balance` optional in type `RawAccount` and use a default of 0 and 0 in `setAccount`:

```
worldAccount := &worldmock.Account{
  Nonce:           0,
  Balance:         big.NewInt(0),
  BalanceDelta:    big.NewInt(0),
  DeveloperReward: big.NewInt(0),
  Storage:         map[string][]byte{},
  MockWorld: 			 e.scenexec.World,
}
```

And let's remove type `UpdateRawAccount`.